phoenix-core/src/main/java/org/apache/phoenix/log/QueryLogger.java

```diff
@@ -155,6 +172,14 @@ public String getQueryId() {
     
 
     public void sync(Map<String, Map<MetricType, Long>> readMetrics, Map<MetricType, Long> overAllMetrics) {
+        syncBase(readMetrics, overAllMetrics, logLevel);
+    }
+
+    public void syncAudit(Map<String, Map<MetricType, Long>> readMetrics, Map<MetricType, Long> overAllMetrics) {
```

Comment:
Since your calls to this method are often passing `(null, null)`, how about overloading this method to create
```
public void syncAudit() {
  syncAudit(null, null, LogLevel.TRACE);
}
```

And then adopt that new call above. Would also be good to have javadoc (especially to denote the difference between `sync` and `syncAudit`)